### PR TITLE
Fix Link

### PR DESCRIPTION
**Pull request by jss367**
The link to IDE integration didn't work for me so I found what seems to be the right link and added it.

## Steps

- [x] Add yourself to CONTRIBUTORS if you are a new contributor.
- [x] Add a ChangeLog entry describing what your PR does.
- [x] If it's a new feature or an important bug fix, add a What's New entry in `doc/whatsnew/<current release.rst>`.
- [x] Write a good description on what the PR does.

## Type of Changes

|   | Type |
| ------------- | ------------- |
| ✓  | :scroll: Docs |

### CODE REVIEW

File: doc/faq.rst [hunk 1]
```diff
@@ -112,7 +112,7 @@ For example::
 3.4 I'd rather not run Pylint from the command line. Can I integrate it with my editor?
 ---------------------------------------------------------------------------------------
 
-Much probably. Read :ref:`ide-integration`
+Much probably. Read http://pylint.pycqa.org/en/latest/user_guide/ide-integration.html
```
Comment (hippo91) [inline doc/faq.rst:115]: This link is not a `reStructuredText` link. By the way the original link seems to be correct.

**Review by hippo91 — CHANGES_REQUESTED** [review]
@jss367 thanks for submitting this PR. However i don't understand the need.
When visiting the following address: http://pylint.pycqa.org/en/2.6/faq.html the link seems to be functional.

**Comment by jss367** [comment]
@hippo91 Maybe I'm editing the wrong page. When I go to https://docs.pylint.org/en/1.6.0/faq.html and click on the link I get this error:

image

I was trying to fix that.

**Comment by hippo91** [comment]
@jss367 the link is indeed broken, as all other links constructed with the `ref` keyword. Moreover when building doc locally (running `python -m tox -edocs` everything is fine and the link is not broken. I don't think the solution is to hard code the broken links however i have no idea where this bug may come from.

**Comment by hippo91** [comment]
@jss367 i'm sorry but i just realized that you were looking at a very old version doc. Please use https://pylint.pycqa.org/ instead. All the links seem to be ok in it.

**Pull request closed without merging**